osgeo_importer/inspectors.py

```diff
@@ -174,21 +174,25 @@ def describe_fields(self):
         for n in range(0, opened_file.GetLayerCount()):
             layer = opened_file.GetLayer(n)
             layer_name = layer.GetName()
+            try:
+                geometry_type = self.geometry_type(layer.GetGeomType())
```

Comment:
What is the exception that gets thrown here?  The `geometry_type` should handle any errors and return `None` by default.